compose/compose-file/index.md

```diff
@@ -3,16 +3,15 @@ description: Compose file reference
 keywords: fig, composition, compose, docker
 redirect_from:
 - /compose/yml
-- /compose/compose-file-v3/
-title: Compose file version 3 reference
+title: Compose file
 toc_max: 4
 toc_min: 1
 ---
 
 ## Reference and guidelines
 
-These topics describe version 3 of the Compose file format. This is the newest
-version.
+These topics describe the Docker Compose implementation of the Compose format.
+Docker Compose **1.27.0+** implements the format defined by the [Compose Specification](https://github.com/compose-spec/compose-spec/blob/master/spec.md). Previous Docker Compose versions have support for several Compose file formats – 1, 2, 2.x, and 3.x. The Compose specification is an unified 2.x and 3.x file format, aggregating properties accross these formats.
```

Comment:
Can we make sure that the version we're linking to matches the version that's used in docker-compose? (ideally, compose-spec would do releases; also see my other comment).

Also wondering if we could pull the actual spec/docs into this repository, so that we can have actual documentation on the file format (instead of users having to read on GitHub)